projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab.component.ts

```diff
@@ -115,6 +115,17 @@ export class LabComponent implements OnInit, OnDestroy {
         this.lab = data;
         this.setMeta();
         this.labDescription = this.sanitizer.bypassSecurityTrustHtml(this.lab.description);
+        
+        if(this.labDescription !=null){
+                          
+          // var imgTags= this.content.nativeElement.querySelectorAll("img");
```

Comment:
Why is this commented?

Reply:
actually needed to remove it from here and add it in ngAfterview()